package.json

```diff
@@ -34,6 +35,7 @@
     "@ngx-translate/http-loader": "^6.0.0",
     "bootstrap": "^4.5.3",
     "esri-loader": "^3.0.0",
+    "moment": "^2.29.1",
```

Comment:
I think we don't need this.

Reply:
yep - you are right.

Reply:
blin am facut - dar in loc sa se adauga in comitul dat s-a facut comit nou ... sorry

Reply:
Am inchis acel PR care ai deschis gresit.

Reply:
@BaxLi can you remove it, so I can merge?

Reply:
done